Gruntfile.js

```diff
@@ -91,7 +91,7 @@ module.exports = function (grunt) {
             'FileDescription': WINDOWS_APPNAME,
             'InternalName': BASENAME + '.exe',
             'OriginalFilename': BASENAME + '.exe',
-            'LegalCopyright': 'Copyright 2015 Docker Inc. All rights reserved.'
+            'LegalCopyright': 'Copyright 2016 Docker Inc. All rights reserved.'
```

Comment:
http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/